content/reference/rest-api/templates/publish.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Templates: publish'
+weight: 10
+
+menu:
+  main:
+    name: 'publish'
+    identifier: 'rest-api-templates-publish'
+    parent: 'rest-api-templates'
+    pre: 'Publishes the current version of a template.'
+---
+
+Publishes the current state of a template as a new version.
+
+# Request
+
+## HTTP Request
+
+```
+POST https://cawemo.com/api/v1/templates/:id/publish
+```
+
+## Request Body
+
+In the request body, supply a [Templates resource]({{< ref "/reference/rest-api/templates/_index.md#resource-representation" >}}) with the following properties:
+
+<table class="table table-striped">
+ <tr>
+   <th>Property Name</th>
+   <th>Type</th>
+   <th>Description</th>
+ </tr>
+  <tr>
+    <td>version.name (optional)</td>
+    <td>string</td>
+    <td>Name of the version to publish.</td>
```

Comment:
Not sure I understand why would we provide the name of the version to publish?

In Cawemo today only the latest content of a template can be published.

Do we intend to change that?

Reply:
> Not sure I understand why would we provide the name of the version to publish?

Sorry, my bad for writing this in an unclear way - I actually meant to say "Name for the version to publish.". This will be the name for the milestone that is created when publishing.